app/controllers/tax_form_due_dates_controller.rb

```diff
@@ -1,13 +1,28 @@
 class TaxFormDueDatesController < ApplicationController
   before_action :set_tax_form_due_date, only: [:show]
 
-  # GET /tax_form_due_dates
+  # GET /tax_form_due_dates -- params entity_type and jurisdiction -TBD
   def index
-    @tax_form_due_dates = TaxFormDueDate.all
+    search = params[:searchkey]
```

Comment:
naming... typically names in ruby are name_name not namename

Reply:
change naming per ruby way